### PR TITLE
Update Code - Proof_Of_ContributionSnapshot5.md

### PR DESCRIPTION
**Pull request by ryanleung**
Completed Windows and Mac builds and packaging for moragents under the guidance of @LachsBagel (i am artfuljars)

### CODE REVIEW

File: Contributions/Code - Proof_Of_ContributionSnapshot4.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ Please include the "pipe" `|` symbol correctly as seen above so your contributio
  4 | 0xA6D60da9ed25509092bA912b523F812E109F3C9c | Devorce99 | PWA: new pages and chat integration | 20000 | 3 |  |  |  |  |  
  4 | 0xe36B28c72C1F7f8a4D87cA5d64B63233875173a4 | xfactor20 | https://docs\.google\.com/document/d/179Hfe\_CUICLBBUY00WnlJu1KSmggkclH\_5XEeQDphM0/edit | 10000 | 3 |  |  |  |  |  
  4 | 0xae855e324087A34E96bB9127d2e17c1A12873020 | openempires | \[MRC31 Thread\]\(https://discord\.com/channels/1151741790408429580/1228539984844165140\) | 5000 | 8 |  |  |  |  |  
- 4 | 0xC3B82270Db1b77B4bE28a83d0963e02c38A9d13f | ryanleung | https://docs\.google\.com/document/d/196\-ew97\-Z436c9xWBtNuWC6ngnaYs6QOfxoGRulLYrY/edit | 20000 | 8 |  |  |  |  |  
+ 4 | 0xC3B82270Db1b77B4bE28a83d0963e02c38A9d13f | artfuljars | https://docs\.google\.com/document/d/196\-ew97\-Z436c9xWBtNuWC6ngnaYs6QOfxoGRulLYrY/edit | 20000 | 8 |  |  |  |  |
```
Comment (ryanleung) [inline Contributions/Code - Proof_Of_ContributionSnapshot4.md:117]: just changing my name here

Comment (betterbrand) [reply Contributions/Code - Proof_Of_ContributionSnapshot4.md:117]: for smart agents?  Should be in that MRI

Comment (ryanleung) [reply Contributions/Code - Proof_Of_ContributionSnapshot4.md:117]: sorry, i mean that i want to change my name from ryanleung -> artfuljars for anonymity! the rest of this line is untouched

Comment (ryanleung) [reply Contributions/Code - Proof_Of_ContributionSnapshot4.md:117]: maybe i can just do this in a separate PR

**Pull request merged**